components/camel-avro-rpc/camel-avro-rpc-component/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the convention in Camel is to have the directory name same as artifactId.